docs/guides/improve-processes-with-optimize.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: improve-processes-with-optimize
 title: Improve processes with Optimize
-sidebar_label: Improve processes with Optimize
```

Comment:
Non-blocking - wasn't "improve" a keyword associated with Optimize? Would you want to adjust the page title, too?

Reply:
CC @RomanJRW @felix-mueller Do you have opinions here on keywords used with Optimize?

Reply:
Not from my side, I think its a PM question predominantly

Reply:
Merging this PR but following up in ask-optimize on preferred keywords 👍